open_api_framework/admin.py

```diff
@@ -0,0 +1,42 @@
+from importlib import import_module
+
+from django.conf import settings
+from django.contrib import admin
+
+from sessionprofile.models import SessionProfile
+
+
+@admin.register(SessionProfile)
+class SessionProfileAdmin(admin.ModelAdmin):
+    list_display = ["session_key", "user", "exists"]
+
+    @property
+    def SessionStore(self):
```

Comment:
I originally set this in the `_init_` but that is not called everytime I ran a test and the cached version always used the database backend.

Reply:
Probably because the pytest fixtures change the `SESSION_ENGINE` after the `__init__` is done, I'm not sure if there is way around this tbh

Reply:
You can just use `override_settings` and it works as usual :/